examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -63,35 +63,38 @@ class _BottomAppBarDemoState extends State<BottomAppBarDemo> {
 
   // FAB Position
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndDocked = _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabEndDocked =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Attached - End',
     label: 'floating action button is docked at the end of the bottom app bar',
     value: FloatingActionButtonLocation.endDocked,
   );
 
-  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterDocked = _ChoiceValue<FloatingActionButtonLocation>(
+  static const _ChoiceValue<FloatingActionButtonLocation> kFabCenterDocked =
+      _ChoiceValue<FloatingActionButtonLocation>(
     title: 'Attached - Center',
```

Comment:
i definitely don't like this. The indentation level should reflect the actual structure of the code, but here line 75 is inside the expression started on line 74 despite the fact that line 74 is indented more. This is IMHO very bad.